sigsep_open-unmix-pytorch_umx.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+category: researchers
+title: Open-Unmix
+summary: Reference implementation for music source separation
+image: sigsep_logo_inria.png
+author: Inria
+tags: [audio, scriptable]
+github-link: https://github.com/sigsep/open-unmix-pytorch
+github-id: sigsep/open-unmix-pytorch
+featured_image_1: sigsep_umx-diagram.png
+featured_image_2: no-image
+accelerator: cuda-optional
+---
+
+
+```bash
+# assuming you have a PyTorch >=1.6.0 installed
+pip install -q torchaudio
+```
+
+```python
+import torch
+
+# loading umxhq four target separator
+separator = torch.hub.load('sigsep/open-unmix-pytorch', 'umxhq')
+
+# audio shape (nb_samples, nb_channels, nb_timesteps)
+# assuming audio is resampled to separator.sample_rate
+audio = torch.rand((1, 2, 100000))
+estimates = separator(audio)
+# estimates.shape = (1, 4, 2, 100000)
+```
+
+### Model Description
+
+__Open-Unmix__ provides ready-to-use models that allow users to separate pop music into four stems: __vocals__, __drums__, __bass__ and the remaining __other__ instruments. The models were pre-trained on the freely available [MUSDB18](https://sigsep.github.io/datasets/musdb.html) dataset.
+
+Open-unmix comprises multiple models that are trained for each particular target.
```

Comment:
do you mean there are multiple ways to do the separation? or one run of the separation used multiple models to do said separation?

Reply:
I think this sentence can be removed to make it clearer